intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -85,10 +110,10 @@ def validate_ip(value):
         ('source.geolocation.cc', 'geo'),
         ('source.geolocation.region', 'region'),
         ('source.geolocation.city', 'city'),
-        # Other known fields  which will go into "extra"
+        # Other known fields which will go into "extra"
+        ('naics', int),
```

Comment:
If NAICS is an empty string this will create trouble.

```
>>> int("")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: ''
```